CMakeLists.txt

```diff
@@ -518,6 +520,40 @@ foreach(target IN LISTS targets)
     link_common_libraries(${target})
 endforeach()
 
+if (fuzzer_conditions)
+  set(FuzzerSrc
+    ${CMAKE_SOURCE_DIR}/src/ripple/conditions/impl/Condition.cpp
+    ${CMAKE_SOURCE_DIR}/src/ripple/conditions/impl/Der.cpp
+    ${CMAKE_SOURCE_DIR}/src/ripple/conditions/impl/Ed25519.cpp
+    ${CMAKE_SOURCE_DIR}/src/ripple/conditions/impl/error.cpp
+    ${CMAKE_SOURCE_DIR}/src/ripple/conditions/impl/Fulfillment.cpp
+    ${CMAKE_SOURCE_DIR}/src/ripple/conditions/impl/PrefixSha256.cpp
+    ${CMAKE_SOURCE_DIR}/src/ripple/conditions/impl/PreimageSha256.cpp
+    ${CMAKE_SOURCE_DIR}/src/ripple/conditions/impl/RsaSha256.cpp
+    ${CMAKE_SOURCE_DIR}/src/ripple/conditions/impl/ThresholdSha256.cpp
+
+    ${CMAKE_SOURCE_DIR}/src/ripple/basics/impl/contract.cpp
+    ${CMAKE_SOURCE_DIR}/src/ripple/basics/impl/Log.cpp
+    ${CMAKE_SOURCE_DIR}/src/ripple/beast/utility/src/beast_Journal.cpp
+    ${CMAKE_SOURCE_DIR}/src/ripple/basics/impl/Time.cpp
+    )
+
+  add_with_props(FuzzerSrc src/ripple/unity/ed25519_donna.c
+    -I"${CMAKE_SOURCE_DIR}/"src/ed25519-donna)
+
+  add_executable(fulfillment_fuzzer  ${FuzzerSrc}
+    ${CMAKE_SOURCE_DIR}/src/fuzzers/Conditions_fuzz_test.cpp)
+  target_link_libraries(fulfillment_fuzzer ${OPENSSL_LIBRARIES} ${SANITIZER_LIBRARIES})
+  link_common_libraries(fulfillment_fuzzer)
+  target_compile_definitions(fulfillment_fuzzer PUBLIC -DFUZZ_TEST_FULFILLMENT)
+
+  add_executable(condition_fuzzer  ${FuzzerSrc}
+    ${CMAKE_SOURCE_DIR}/src/fuzzers/Conditions_fuzz_test.cpp)
+  target_link_libraries(condition_fuzzer ${OPENSSL_LIBRARIES} ${SANITIZER_LIBRARIES})
```

Comment:
I have not used libFuzzer myself, but docs indicate that you still need to link with `libFuzzer.a` - so we might need to add that here ?